src/ReleasableArray2D.cpp

```diff
@@ -71,6 +71,19 @@ namespace ZividPython
             .def_buffer(pointCloudDataBuffer<NativeType, WrapperType, 4>);
     }
 
+    void wrapClass(pybind11::class_<ReleasableArray2D<Zivid::ColorBGRA>> pyClass)
+    {
+        using WrapperType = uint8_t;
+        using NativeType = Zivid::ColorBGRA;
+        static_assert(std::is_same_v<WrapperType, decltype(NativeType::g)>);
+        static_assert(std::is_same_v<WrapperType, decltype(NativeType::g)>);
```

Comment:
Duplicate line. First one should probably be "b"?

Reply:
good catch